src/window/empire.c

```diff
@@ -266,6 +266,9 @@ static px_point trade_amount_px_offsets[5] = {
     { 4, 6 },
 };
 
+//values for drawing animated dots
+static struct {int x, y, sea;} anim_dots[128]; //This limits number of trade routes, increase if someone complains
```

Comment:
write structs with each arg in new line,
```suggestion
static struct {
int x;
int y; 
int is_sea;
} anim_dots[128]; //This limits number of trade routes
```
Is it the limit of trade routes or limit of dots, because i've made that mistake before?

Reply:
It's a limit on trade routes and I wrote it like this because I saw struct { int x, y; } dot_pos[1024];